tutorial/src/androidTest/kotlin/com/kaspersky/kaspresso/tutorial/kautomator_test/UiSimpleTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.kaspersky.kaspresso.tutorial.kautomator_test
+
+import android.Manifest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.rule.GrantPermissionRule
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import com.kaspersky.kaspresso.tutorial.MainActivity
+import com.kaspersky.kaspresso.tutorial.R
+import com.kaspersky.kaspresso.tutorial.kautomator_screen.UiMainScreen
+import com.kaspersky.kaspresso.tutorial.kautomator_screen.UiSimpleScreen
+import io.github.kakaocup.kakao.common.utilities.getResourceString
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ * In this example you can see a test tuned by default Kaspresso configuration.
+ * When you start the test you can see output of default Kaspresso interceptors:
+ * - a lot of useful logs
+ * - failure handling
+ * - screenshots in the device
+ * Also you can see the test DSL simplifying the writing of any test
+ */
+class UiSimpleTest : TestCase() {
+
+    @get:Rule
+    val runtimePermissionRule: GrantPermissionRule = GrantPermissionRule.grant(
+        Manifest.permission.WRITE_EXTERNAL_STORAGE,
+        Manifest.permission.READ_EXTERNAL_STORAGE
+    )
+
+    @get:Rule
+    val activityTestRule = ActivityTestRule(MainActivity::class.java, true, false)
```

Comment:
Can we use ```activityScenario<MainActivity>``` here? And remove ```launchActivity``` from the test

Reply:
Thanks, it makes sense